test/Staker.invariants.t.sol

```diff
@@ -77,6 +77,13 @@ contract StakerInvariants is Test {
     );
   }
 
+  function invariant_Sum_of_unclaimed_reward_should_be_less_than_or_equal_to_total_rewards() public {
```

Comment:
We should consider extending the last part of the name for clarity: `_total_rewards_balance()`. Saying simply "total rewards" makes it sound like it could mean all rewards ever.

Reply:
renamed in https://github.com/withtally/staker/pull/80/commits/8d2844eac4d8069009d2dd67674b5c8226133105